package.json

```diff
@@ -44,10 +44,11 @@
     "nightmare": "^2.10.0",
     "nightmare-upload": "^0.1.1",
     "node-dev": "^3.1.3",
-    "nyc": "^11.2.1",
+    "nyc": "^11.4.0",
     "portscanner": "^2.1.1",
     "sinon": "^3.2.1",
     "sinon-chai": "^2.13.0",
+    "sonar-scanner": "^1.0.1",
```

Comment:
the version looks a little old, is there any reason for that? See latest version here -> https://github.com/bcaudan/node-sonar-scanner/blob/master/package.json#L3

Reply:
Another thing, I would suggest you to merge or rebase from master to bring in latest changes.

Reply:
Already merged lastest from master to bring this branch up to date.  However i will wait @paulgain to review this